### PR TITLE
[Modular] Skrell Update!

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
Honestly skrell being just a human re-skin makes me deeply angry!
I still need to make the *warble sound which will take less than 10 minutes.
Everything works just need to test the language again

## Why It's Good For The Game
I mean different species gameplay is fun!
I just mixed a lot of codebase skrell functions!
Aurora:
*Skrell have a haemocyanin-based circulatory system, meaning that their blood is instead bound to copper atoms and not iron atoms. Skrell require copper supplements to alleviate bloodloss symptoms.*
*Due to their advanced vestibular system, Skrell maintain a good balance and never slip.*
*skrells have long contact with slimes making they understand they language*
Baystation And Paradise:
*Despite an extremely low resistance to alcohol, Skrell have otherwise developed a high tolerance to a large amount of chemical substances, a result of their homeworld's flora being generally poisonous as a defence mechanism. Consequently, skrellian food is generally too harmful for human consumption.*
*their eyes are larger than humans, occupying half their face and entirely black when observed from the outside. Their size and position, being further apart than on human faces, give the Skrell a broader field of view.*
image

https://user-images.githubusercontent.com/67030229/113063304-77200180-918b-11eb-80f3-8eb599e0d66d.mp4

## Changelog
:cl:
add: Added the Skrell Language
add: Added new organs to skrell (Sprites made on Paradise). The lungs of skrells can support a high amount of toxin in the air but are extremely frail to low and high temperatures. Even with 50 degrees, they will start taking damage. And Skrellian eyes work like Moth. They are weak against light. Skrellian Liver can support a high amount of toxin but they are extremely weak against alcohol.
balance: General Rebalance on skrell
balance: Skrell can't slip.
balance: Skrell blood is now copper
balance: Skrell takes 1.70 more brute and 0.60 less burns and their body temperature is higher than most other species.
add: Warble emote 
balance: Skrell can speak Slime
/:cl:

### CODE REVIEW

**Comment by Arctaisia** [comment]
Perfection

**Comment by FlamingLily** [comment]
Okay, while on one hand **fuck yeah Skrell**, on the other, IIRC paradise's (and I presume Bay's) Skrell language is /TG/s slime language, so that might be a lil' confusing. It's probably fine though.

**Comment by Guidesu** [comment]
need fix the skrell language then its all guchi

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/skrell.dm [hunk 1]
```diff
@@ -2,15 +2,78 @@
...
+	icon = 'modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/skrell_organ.dmi'
+	icon_state = "brain2"
+
+/obj/item/organ/eyes/skrell
+	name = "amphibian eyes"
+	desc = "Large black orbs."
+	icon = 'modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/skrell_organ.dmi'
+	icon_state = "eyes"
+	flash_protect = FLASH_PROTECTION_SENSITIVE
+/obj/item/organ/lungs/skrell
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/skrell.dm:75]: Bit of better formatting here, have a gap

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/skrell.dm [hunk 2]
```diff
@@ -2,15 +2,78 @@
...
+
+	hot_message = "You can't stand the searing heat with every breath you take!"
+	heat_level_1_threshold = 318
+	heat_level_2_threshold = 348
+	heat_level_3_threshold = 1000
+	heat_level_1_damage = HEAT_GAS_DAMAGE_LEVEL_2
+	heat_level_2_damage = HEAT_GAS_DAMAGE_LEVEL_2
+	heat_level_3_damage = HEAT_GAS_DAMAGE_LEVEL_3
+	heat_damage_type = BURN
+/obj/item/organ/liver/skrell
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/skrell.dm:101]: same here

**Comment by Guidesu** [comment]
image
My bad my VSC show a little gap there

**Review by Gandalf2k15 — APPROVED** [review]

**Comment by Guidesu** [comment]
dang i still need to fix the skrell bug

**Comment by Guidesu** [comment]
fixed

**Comment by KasparoVy** [comment]
Wicked! Squid squad.

**Pull request merged**